aws/cloudformation/standalone/gen_ai_curriculum/monitoring/dashboard/createDashboard.ts

```diff
@@ -80,7 +82,6 @@ function createDashboard(environment = 'production'): Dashboard {
       createBrowserLatencyComparisonGraph(browsers, environment),
       createActiveJobGraph(environment),
       createLatencyComparisonGraph(modelDescriptions, environment),
-      createBrowserLatencyByModelGraph(modelDescriptions, environment),
```

Comment:
Curious why you removed this.

Reply:
This was just kind of overly complex to construct without SEARCH queries and we already had the same data displayed in different views, so I just removed it for simplicity's sake 😅